config/services.yml

```diff
@@ -4,3 +4,11 @@ services:
         public: true
         autowire: true
         tags: ['controller.service_arguments']
+
+    my.secure_handler:
+      class: \PhpList\RestBundle\ViewHandler\SecuredViewHandler
+
+    my.secure_view_handler:
+      parent: fos_rest.view_handler.default
+      calls:
+       - ['registerHandler', [ 'json', ["@my.secure_handler", 'createResponse'] ] ]
```

Comment:
Please let's unify this to always use single quotes unless the double quotes are required here.